### PR TITLE
Mturk max time

### PR DESCRIPTION
**Pull request by marcelhollerbach**
- Adds automaticall hit extending to mturk
- Adds style to the mturk worker-ui
- Sets the maxassignment to 100

### CODE REVIEW

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/mturk/HitExtender.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    }
+
+    /**
+     * Add a new hit to the list of hits
+     *
+     * @param id hit id to add
+     */
+    public void addHit(String id) {
+        synchronized (hitIds) {
+            hitIds.remove(id);
```
Comment (lckr) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/mturk/HitExtender.java]: Is this supposed to be remove(id) ?

Comment (marcelhollerbach) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/mturk/HitExtender.java]: F***, you are right will fix it later.

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Main.java [hunk 2]
```diff
@@ -257,13 +258,17 @@ private static void initLogLevel(Config config) {
             Platform platformInstance;
             switch (platform.type) {
                 case "mturk":
+                    MturkPlatform mturkPlatform;
                     //FIXME remove the sandbox url - but I am to paranoid
-                    platformInstance = new MturkPlatform(platform.user,
+                    platformInstance = mturkPlatform = new MturkPlatform(platform.user,
```
Comment (LeanderK) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Main.java:266]: this is valid java? o.O

Comment (kelunik) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Main.java:266]: sure.

Comment (marcelhollerbach) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Main.java:266]: I added the feature tonight to the jdk and sneaked the new binary with my hexxor skills on your laptop. Better protect yourself. 
tldr; yes.
